hoomd/snapshot.py

```diff
@@ -288,6 +288,38 @@ def constraints(self):
         else:
             raise RuntimeError('Snapshot data is only present on rank 0')
 
+    @property
+    def mpcd(self):
+        """MPCD data.
+
+        Attributes:
+            mpcd.N (int): Number of MPCD particles.
+
+            mpcd.position ((*N*, 3) `numpy.ndarray` of `float`):
+                Particle position :math:`[\\mathrm{length}]`.
+
+            mpcd.velocity ((*N*, 3) `numpy.ndarray` of `float`):
+                Particle velocity :math:`[\\mathrm{velocity}]`.
+
+            mpcd.types (list[str]):
+                Names of the particle types.
+
+            mpcd.typeid ((*N*, ) `numpy.ndarray` of ``uint32``):
+                Particle type id.
+
+            mpcd.mass (float): Particle mass.
+
+        Note:
+            Set ``N`` to change the size of the arrays.
+        """
```

Comment:
Maybe use `hoomd.version` to check for MPCD support and error out first, that will simplify what comes next.

Reply:
```
if not hoomd.version.mpcd_built:
    raise ...
```